### DISCUSSION TITLE
Difference between Android and iOS 3D model

### DISCUSSION THREAD

**Discussion by divclass1** [google/model-viewer#4915] [Q&A]
Hi,

I have a lamp in AR. When I view it from the top the part with the circles is visible. But when I view it from the bottom, it's not visible. This issue only happens on iOS. On Android it's visible from the top and bottom. How can I fix this?

Thanks in advance!!
![IMG_4043](https://github.com/user-attachments/assets/e6adc2c6-419f-4bd7-a915-b35abfc876ff)
![IMG_4042](https://github.com/user-attachments/assets/4564b852-ef44-48ae-adfc-54e33c7ebbc9)

**Comment by elalish** [comment]
Welcome to the joys of USDZ - Apple's unspec'ed 3D format. I don't actually know if it even is fixable; USDZ is missing a lot of important 3D features. @AdaRoseCannon in case Apple has any insight, and to politely prod them to support glTF so that users don't need to deal with their files being mangled by conversion.

**Reply by divclass1** [reply]
Thanks for your quick reply.  @elalish  Really appreciate it. Hopefully @AdaRoseCannon can do something about this 😂

**Reply by enzodbspace** [reply]
I'm late to the party, but we've had similar issues with a client, and the problem was the **normals** of the 3D model. If there is a reversed normal ("red" in Blender), iOS doesn't display it on the reversed side (example: the client had a dinner table, where the legs were only visible if you looked at it from the outside, but not from the inside). Funny, but it took hours to figure out the problem.
